tests/data/pretrain/pretrain_config.yaml

```diff
@@ -0,0 +1,55 @@
+data:
+  dataset:
+    ignore_special_tokens: true
+    masking_ratio: 0.8
+    replace_ratio: 0.1
+    select_ratio: 1.0
+  min_len: 2
+  num_train_patients: 100
+  num_val_patients: 20
+  truncation_len: 20
+  val_ratio: 0.2
+logging:
+  level: 20
+  path: ./.test_tmp/logs
+metrics:
+  mlm_loss:
+    _target_: corebehrt.evaluation.metrics.LossAccessor
+    loss_name: loss
+  top1:
+    _target_: corebehrt.evaluation.metrics.PrecisionAtK
+    topk: 1
+  top10:
+    _target_: corebehrt.evaluation.metrics.PrecisionAtK
+    topk: 10
+model:
+  abspos_embeddings: false
+  hidden_size: 96
+  intermediate_size: 64
+  linear: true
+  num_attention_heads: 3
+  num_hidden_layers: 3
+  plusplus: true
```

Comment:
Some of these things (`plusplus`, `linear` and `abspos_embeddings`) has been made default in the model. Lets just remove them here

Reply:
Makes sense, but could probably be done as part of the config documentation issue #46 (as we have to look through the configs then anyways)